.github/workflows/install-xmllint/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Install XMLLint
```

Comment:
I think that convention is to put actions under `.github/actions`

Reply:
d7cb95a1bae5d2ea32e7620edbac4e648eebf403